### ISSUE TITLE
Special function to change status of Function switches

### ISSUE THREAD

**Issue by Zadrob** [FrSkyRC/ETHOS-Feedback-Community#307]
Hello, 
I would appreciate possibility to change status of Function Switches - for example via Special Function -  to change FSW from 1 to 6 and most importantly to switch all OFF (no LED) (and possibly disable them) - imagine situation of more functions (mode Flight modes)  controlled also via normal Switch - in one position it will turn all FSW off, returning of switch other position will return to normal work of FSW.

Thank you 
Karel

**Comment by MIKE-DAILEY** [comment]
Related to #105

**Comment by Cipta-de** [comment]
281 / 5000
Hello,
does the function switch always have to jump back to one when starting?
If I use the function switch otherwise (as a variable) then it is not so advantageous - it would be better (in some applications) if the last active switch were activated again (like the manual rotary 6pos switch)

**Comment by Zadrob** [comment]
Hello 
Whatever. I think, that last FSW could be stored in global variable for example. 
For me the best way would be functions like:

Set FSW 0...6 (0 means no FSW, but with possibility to activate any of them) 
Lock FSW (lock status) / Unlock FSW 
Disable FSW (all off and lock them) 
Enable FSW (with return to last value) 

Karel

**Comment by rburrow87** [comment]
Yes it would be nice to have it remember the last position of the FSW on a per-model basis.

Or have the ability to assign the default starting position per model.

Being able to set it to off (or FSW0) would be nice too so that the light isn't on for models it's not used for.

**Comment by mawzthefinn** [comment]
Last position of the FSW's are remembered on a per module basis, both when switching models and restarting, at least for current (1.4.x) firmware

Off is possible as well, if set to 6pos + off.

The OP's original ask can achieved with VAR mixes for FC flight mixes with correct priority setup, or using Flight Mode priority for Transmitter flight modes.
